src/components/views/rooms/RoomSublist.tsx

```diff
@@ -402,6 +402,7 @@ export default class RoomSublist extends React.Component<IProps, IState> {
 
     private onTagSortChanged = async (sort: SortAlgorithm) => {
         await RoomListStore.instance.setTagSorting(this.props.tagId, sort);
+        this.forceUpdate();
```

Comment:
Fixes the re-rendering problem, https://github.com/matrix-org/matrix-react-sdk/pull/7353#discussion_r769656166

Reply:
👎  for forceUpdate but not the fault of this PR, thanks for making it less buggy!